packages/h5p-server/src/renderers/player.ts

```diff
@@ -11,7 +12,7 @@ export default (model: IPlayerModel): string => `<!doctype html>
     ${model.scripts
         .map((script) => `<script src="${script}"></script>`)
         .join('\n    ')}
-
+    <style>${generateThemeCSS(model.theme)}</style>
```

Comment:
The theme should be optional

Reply:
It is optional. When the theme is not defined it returns an empty string.

Reply:
On second thought I think the way to inject the theme with the global custom style is better, as it doesn‘t require changes in the client and it will also work with the iframe embed type.